modules/running-insights-operator-gather-cli.adoc

```diff
@@ -5,45 +5,102 @@
 +
-Replace `<your_datagather_definition>.yaml` with a configuration file that contains the following parameters:
+[source,yaml]
+----. To create a new `DataGather` operation where all gather operations will run, complete the following YAML specification, and then save your changes.
 +
 [source,yaml]
 ----
-apiVersion: insights.openshift.io/v1alpha1
+
+apiVersion: insights.openshift.io/v1alpha2
 kind: DataGather
 metadata:
-  name: <your_data_gather> <1>
+  name: <your_data_gather>
 spec:
- gatherers: <2>
-   - name: workloads
-     state: Disabled
+# Gatherers configuration
+  gatherers:
+    mode: All # Options: All, None, Custom
 ----
 +
---
-<1> Under *metadata*, replace `<your_data_gather>` with a unique name for the gather operation.
-<2> Under *gatherers*, specify any individual gather operations that you intend to disable. In the example provided, `workloads` is the only data gather operation that is disabled and all of the other default operations are set to run.
-When the `spec` parameter is empty, all of the default gather operations run.
---
-
 [IMPORTANT]
 ====
-Do not add a prefix of `periodic-gathering-` to the name of your gather operation because this string is reserved for other administrative operations and might impact the intended gather operation.
+* The name you specify for your gather operation, `<your_data_gather>`, must be unique and must not include a prefix of `periodic-gathering-` because this string is reserved for other administrative operations and might impact the intended gather operation.
+* If the `spec` of `DataGather` CRD is undefined, the default Insights Operator data collection job will run. This means that all gather operations will run, the collected data will be unobfuscated and the archive file will not be retained.
 ====
++
+. Optional: To customize the data gather operation, you can configure any of the following options in your `DataGather` YAML file:
+* To disable specific gatherers, change the value of `mode` to *Custom*, and then specify the individual gatherer that you intend to disable. For example, to disable the workload gatherer, add the following example:
++
+[source,yaml]
+----
+apiVersion: insights.openshift.io/v1alpha2
+kind: DataGather
+metadata:
+  name: <your_data_gather> 
+spec:
+    # Gatherers configuration
+  gatherers:
+    mode: Custom  # Options: All, None, Custom
+    custom:
+      configs:
+        # Essential cluster configuration gatherers
+        - name: clusterconfig/authentication
+          state: Enabled
+        - name: clusterconfig/clusteroperators
+          state: Enabled
+        - name: workloads
+          state: Disabled
+----
+* To enable persistent storage to retain the data archive file and history for up to the last 10 data gathering jobs, define the `storage` specification. Set *type* to `PersistentVolume`, and define the volume's `mountPath` and `name`, as outlined in the following example:
```

Comment:
I would highlight here that the `name` needs to reference existing PersistentVolumeClaim in the `openshift-insights` namespace.